C++/0. Setup.md

```diff
@@ -2,17 +2,13 @@
 
 ## Start
 
-So, in C++ files, there are some things you need to add at the start of the document. `#include <iostream>` includes a package in C++ which lets you use `cout` and `cin` which are the keywords for character output and character input. In some beginner programmer's programs, you may see `using namespace std;` at the starting too, which lets you use a lot of functions without needing to write `std::functionName` all the time. This is useful, but it sometimes includes functions you don't need and you didn't know were there, resulting with you creating a function with the same name which is used in a function in the namespace. That is why an alternative is `using std::functionName;` which only lets you use that specific function without needing to write `std::functionName` all the time. Finally the `int main() {}` is where you will put your code. Your code goes inside the curly braces `{}`. This creates a function called main, which is automatically run when you run the file, and the code inside the braces is what will be run.
+So, in C++ files, there are some things you need to add at the start of the document. `#include <iostream>` includes a package in C++ which lets you use `cout` and `cin` which are the keywords for character output and character input. In some beginner programmer's programs, you may see `using namespace std;` at the starting too, which lets you use a lot of functions without needing to write `std::functionName` all the time. This is useful, but it sometimes includes functions you don't need and you didn't know were there, resulting with you creating a function with the same name which is used in a function in the namespace. Finally the `int main() {}` is where you will put your code. Your code goes inside the curly braces `{}`. This creates a function called main, which is automatically run when you run the file, and the code inside the braces is what will be run.
 
 ### Code
 
 ```cpp
 #include <iostream>
 
-using std::cout;
-using std::cin;
-using std::string;
```

Comment:
umm isn't this fine? I didn't write `using namespace std;`. This only imports the specific file. It is easier than writing std:: before everything...